Deployment/README.md

```diff
@@ -0,0 +1,73 @@
+# How to run locally
+There are three ways to run the code in this repository locally:
+1. Mocking out the whole backend for just working on the frontend
+2. Running with Docker Compose (the setup is a bit simpler than the whole Kubernetes environment)
+3. Running in a local Kubernetes cluster to most accurately simulate the runtime environment.
+
+## 1. Running with a mocked backend
+> The files and scripts referenced in this section is in the `/Deployment/Frontend` directory in this repository.
+
+Install and run the express server:
+```shell
+yarn
+yarn start
+```
+
+Start the frontend code in `Source/Login/Web`, navigate to http://localhost:8080.
+
+Select a provider to log in, then a tenant, and you should be presented with a page saying you are logged in. The page also has a link to restart the login process, or to view the error page in the frontend.
+
+To test out with different sets of providers or tenants, you can modify the `identityProviders` and `availableTenants` variables on the top of the `index.js` file.
+
+> Note: no special handling is required for a case where a single tenant is available for a user, this is handled by the backend which would jump over the select tenant page.
+
+## 2. Running with Docker Compose
+> The files and scripts referenced in this section is in the `/Deployment/Development` directory in this repository.
+> It also requires a local installation of Go.
+
+##### Configuring the network
+First, set the IP of your host machine (reachable from a Docker container) to the `HOST_IP` in the `.env` file.
```

Comment:
```suggestion
First, set the IP of your host machine (reachable from a Docker container) to the `HOST_IP` in the `.env` file. You can find this on Linux with `ip` command and it's `inet` line:
```shell
$ ip addr show docker0
```

The suggested `192.168.49.1` didn't work for my Ubuntu 20.04, I followed [this guide](https://stackoverflow.com/a/33103501) on checking what it would be and it was `172.17.0.1` for me so I added some extra lines here.

Reply:
Thanks! The suggestion was missing the closing triple quotes. I added it as a note/quote in a separate commit.